src/dune_pkg/lock_dir.mli

```diff
@@ -71,7 +71,12 @@ module Write_disk : sig
   type lock_dir := t
   type t
 
-  val prepare : lock_dir_path:Path.Source.t -> lock_dir -> t
+  val prepare
+    :  lock_dir_path:Path.Source.t
+    -> files:(Path.t * Path.Local.t) Package_name.Map.Multi.t
```

Comment:
There is some redundant information here since the `Path.t` contains the local path too. Perhaps it would be better to just keep a root and a local path.

Reply:
I don't think it matters if there's a redundancy. However, I do think that the tuple is making it harder to read than necessary. Can you add a proper record to represent this?